dropwizard-client/src/main/java/io/dropwizard/client/DropwizardApacheConnector.java

```diff
@@ -125,9 +128,43 @@ private HttpUriRequest buildApacheRequest(ClientRequest jerseyRequest) {
             }
             builder.addHeader(headerName, jerseyRequest.getHeaderString(headerName));
         }
+
+        Optional<RequestConfig> requestConfig = addJerseyRequestConfig(jerseyRequest.getConfiguration());
+        if (requestConfig.isPresent()) {
+            builder.setConfig(requestConfig.get());
+        }
+
         return builder.build();
     }
 
+    private Optional<RequestConfig> addJerseyRequestConfig(Configuration configuration) {
+        RequestConfig.Builder requestConfig = RequestConfig.copy(client.getDefaultRequestConfig());
```

Comment:
I think it's better to make a copy lazily, only in a case if there changes in the configuration.